husky_components/navigation/main.js

```diff
@@ -253,7 +249,11 @@ define(function() {
             // load Data
             if (!!this.options.url) {
                 this.sandbox.util.load(this.options.url, null, 'json')
-                    .then(this.render.bind(this))
+                    .then(function(data) {
+                        this.options.data = data;
```

Comment:
Was this already `this.options.data`? Because internal data should not be stored in this object, since it contains options being passed from the outside